### PR TITLE
Set default HTTPOption to Enabled for conformance test

### PR DESCRIPTION
**Pull request by ZhiminXiang**

### CODE REVIEW

File: test/conformance/ingress/util.go [hunk 1]
```diff
@@ -686,6 +686,8 @@ func CreateIngress(ctx context.Context, t *testing.T, clients *test.Clients, spe
 		t.Fatalf("Invalid ingress %q: %v", ingName, err)
 	}
 
+	setDefaultsForTest(ing)
```
Comment (nak3) [inline test/conformance/ingress/util.go:689]: Do we have a plan to add a default value to `ing.Spec.HTTPOption` by `SetDefaults()`?
 If so, we should add `TODO:` comment.

Comment (ZhiminXiang) [reply test/conformance/ingress/util.go:690]: Yes. I think we should set the default value in SetDefaults in the future. I added a TODO here.

**Pull request merged**